internal/types/handlers.go

```diff
@@ -0,0 +1,9 @@
+package types
+
+import "net/http"
+
+type Handlers struct {
+	UpdateHandler   func(http.ResponseWriter, *http.Request)
```

Comment:
[CLARIFICATION] What did you want to do it here? It seems you can avoid it.

Reply:
<img width="570" alt="image" src="https://github.com/user-attachments/assets/a4807d52-b90e-4761-a5d5-3aa9eec11ff6">

to use as an object and pass to the router, maybe there is a better way

Reply:
I've changed the structure, please review